packages/gatsby-theme-vtex/src/components/Minicart/Drawer/Delete.tsx

```diff
@@ -0,0 +1,41 @@
+import { Button, Spinner } from '@vtex/store-ui'
+import React, { useState } from 'react'
+
+import { useUpdateItems } from './useUpdateItems'
+
+type Props = {
+  index: number
+  variant?: string
+}
+
+const MinicartDelete = (props: Props) => {
+  const [isLoading, setIsLoading] = useState(false)
+  const updateItems = useUpdateItems(props.index)
+  const useDelete = {
+    variant: `${props.variant}.delete`,
+    onClick: async () => {
```

Comment:
No need for it to be async, right?

Just: `updateItems(0)`

Reply:
indeed